pages/api/children/[childId]/accounts/[accountId]/transactions.js

```diff
@@ -0,0 +1,47 @@
+import connectDB from "../../../../_db/connect-db";
+import {Child} from "../../../../_db/models/Child";
+
+async function handler(req, res) {
+  switch (req.method) {
+    // case "GET":
+    //   try {
+    //     const child = await Child.findById(req.query.childId);
+    //     console.log(req.query.childId);
+    //     if (child) {
+    //       return res.status(200).json(child.accounts);
+    //     } else {
+    //       return res.status(500).json({error: "user not found"});
+    //     }
+    //   } catch {
+    //     console.log("error");
+    //   }
+    //   break;
```

Comment:
is this intentionally commented out?

Reply:
Yes, I haven't worked on the GET method yet, that's the next step.